### ISSUE TITLE
Docker implementation

### ISSUE THREAD

**Issue by katseyres2** [carambole25/webLogScan#1]
You could develop docker containerization to create an easy service installation.

**Comment by carambole25** [comment]
Yes it's a good idea but what scares me is the -bs option (ban scan with ufw) if the user wants to use this function he will have to give the docker access to his host network with this: --network host --privileged

I don't know if this is a problem I don't know enough docker but people on the internet say it's a bit dangerous.

But I'm going to do it anyway.

**Comment by katseyres2** [comment]
If you're using ufw file, you could mount it as a volume into the container.

**Comment by carambole25** [comment]
what do you think about this ?
```
$ sudo docker run --rm carambole25/web-log-scan:latest
usage: wls.py [-h] [-ss] [-is] [-bs] path

$ sudo docker run --rm -v "$(pwd)/logpath/your_log_file:/logs/your_log_file" carambole25/web-log-scan:latest -ss /logs/your_log_file

$ sudo docker run --rm -v "$(pwd)/logpath/your_log_file:/logs/your_log_file" carambole25/web-log-scan:latest -is /logs/your_log_file

$ sudo docker run --rm --network host --privileged -v "$(pwd)/logpath/your_log_file:/logs/your_log_file" carambole25/web-log-scan:latest -bs /logs/your_log_file
``

**Comment by carambole25** [comment]
if i don't this is what append : 
`$ sudo docker run --rm --network host -v "$(pwd)/logpath/your_log_file:/logs/your_log_file" carambole25/web-log-scan:latest -bs /logs/your_log_file
ERROR: problem running ufw-init
iptables-restore v1.8.9 (nf_tables): Could not fetch rule set generation id: Permission denied (you must be root)
iptables-restore v1.8.9 (nf_tables): Could not fetch rule set generation id: Permission denied (you must be root)
iptables-restore v1.8.9 (nf_tables): Could not fetch rule set generation id: Permission denied (you must be root)
iptables-restore v1.8.9 (nf_tables): Could not fetch rule set generation id: Permission denied (you must be root)
iptables-restore v1.8.9 (nf_tables): Could not fetch rule set generation id: Permission denied (you must be root)
iptables-restore v1.8.9 (nf_tables): Could not fetch rule set generation id: Permission denied (you must be root)
iptables-restore v1.8.9 (nf_tables): Could not fetch rule set generation id: Permission denied (you must be root)
iptables-restore v1.8.9 (nf_tables): Could not fetch rule set generation id: Permission denied (you must be root)
iptables-restore v1.8.9 (nf_tables): Could not fetch rule set generation id: Permission denied (you must be root)
iptables-restore v1.8.9 (nf_tables): Could not fetch rule set generation id: Permission denied (you must be root)
/lib/ufw/ufw-init: 387: sysctl: not found
/lib/ufw/ufw-init: 387: sysctl: not found

Problem loading ipv6 (skipping)
Problem running '/etc/ufw/before.rules'
Problem running '/etc/ufw/after.rules'
Problem running '/etc/ufw/user.rules'

WARN: initcaps
[Errno 2] iptables v1.8.9 (nf_tables): Could not fetch rule set generation id: Permission denied (you must be root)

Rules updated
Firewall not enabled (skipping reload)
['959.156.855.254'] (it's a random ip) 
`
